env-check.sh

```diff
@@ -24,12 +27,22 @@ func_check_python_pkg(){
     fi
 }
 
+func_check_file(){
+    if [ -e $1 ]; then
+        return
+    else
```

Comment:
No need for the `else` and extra indentation

Reply:
I mean:
```shell
if [ -e "$1" ]; then
   return 0
fi
out_str=...
check_res=1

Reply:
agree with @marc-hb, it's the same style in kernel-land. When you do a return the following else is not useful.

Reply:
> When you do a return the following else is not useful.

https://www.google.com/search?q=chad+versace+early+return+perfect+ice+cream

https://en.wikipedia.org/wiki/Cyclomatic_complexity